### ISSUE TITLE
TILMedia simulation process crashes

### ISSUE THREAD

**Issue by HSchatzTLK** [OpenModelica/OpenModelica#8546]
### Description
Simulation process of TILMedia testers crashes.

### Steps to reproduce
Start OMEdit (Windows 64-bit)
Open TILMedia (https://github.com/TLK-Thermo/TILMediaClaRa)
Simulate one of the models in the "Testers" package (e.g. TILMedia.Testers.TestLiquid, default simulation setup was used)

### Expected behavior
Successful simulation. According to the [OM tests](https://libraries.openmodelica.org/branches/master/TILMedia/TILMedia.html) the models simulate successfully on Linux.

### Screenshots
![image](https://user-images.githubusercontent.com/97731319/154045736-245a061a-8552-4e09-8a5e-dab6f02ffae6.png)

### Version and OS
 - OpenModelica Version: omc v1.19.0-dev-586-gbf2fb176ac-64bit
 - OS: Windows 10
 - TILMedia ClaRa 1.7.0

**Comment by mahge** [comment]
This has now been fixed with #8552. The model in the ticket compiles and simulates successfully. Please check and reopen if it is not working on your side.

**Comment by AnHeuermann** [comment]
Still not solved in OMEdit.

I will have a look and try @adrpo's solution.

**Comment by mahge** [comment]
Yes unfortunately it is not solved in OMEdit. However, #8552 has fixed the API functions `simulate`, `linearize` and `optimize`. They should work just fine now. 

I considered writing a script that is generic and accepts parameters (similar to `compile.bat`) OR generating one on a model per model basis (similar to how the CPP runtime launches simulation). They both have their advantages. In the end though it seemed enough to set the PATH directly. I did not think of the fact that OMEdit does not use `simulate`. 

I am looking at OMEdit to figure out which APIs it calls. I am guessing it starts with `buildModel` and then manually calls the executable to simulate it. In that case we have two options. 

As a first option we can generate the scripts (generic one or for each model). Second option is to repeat the same process as #8522 in OMEdit. `buildModel` should now give you back the necessary directories needed for locating the DLLs.

**Comment by AnHeuermann** [comment]
Do we have any name clashes in the libraries?
Otherwise we could try to always have all loaded Modelica libraries inside `PATH`.

Then it should just be adding
```
C:\Users\username\AppData\Roaming\.openmodelica\libraries\Buildings\Resources\Library\win64;
C:\Users\username\AppData\Roaming\.openmodelica\libraries\Buildings\Resources\Library;
```
for all libraries in OMEdit to `PATH` environment variable and then call the executable in a shell with that updated variable.

**Comment by mahge** [comment]
Might work but note that the paths come from annotations. They are not computed automatically by omc. right? 

We need to add any paths users have added in there. It might not always be these "standard" paths.

**Comment by mahge** [comment]
This should now be (this time for real) fixed in #8572. If you pick up tomorrow's nightly build, the model should work fine from OMEdit as well. I am going to close this again. Please open it if there are more issues.
